CMakeLists.txt

```diff
@@ -111,6 +111,6 @@ set(ignoreMe "${SKBUILD}")
 add_subdirectory(cpp)
 add_subdirectory(freud)
 
-if(_using_conda)
+if(_using_conda OR DEFINED ENV{CIBUILDWHEEL})
   set_target_properties(libfreud PROPERTIES INSTALL_RPATH_USE_LINK_PATH True)
```

Comment:
With this change, I don't think the above `LD_LIBRARY_PATH` setting is necessary.